packages/state-transition/src/util/epochShuffling.ts

```diff
@@ -109,9 +110,46 @@ export function computeEpochShuffling(
 ): EpochShuffling {
   const activeValidatorCount = activeIndices.length;
 
-  const shuffling = activeIndices.slice();
   const seed = getSeed(state, epoch, DOMAIN_BEACON_ATTESTER);
-  unshuffleList(shuffling, seed);
+  const shuffling = unshuffleList(activeIndices, seed, SHUFFLE_ROUND_COUNT);
+
+  const committeesPerSlot = computeCommitteeCount(activeValidatorCount);
```

Comment:
deduplicate common code of `computeEpochShuffling` and `computeEpochShufflingAsync` if possible

Reply:
Good idea.  Updated.